pkg/oc/originpolymorphichelpers/updatepodspec.go

```diff
@@ -3,78 +3,24 @@ package originpolymorphichelpers
 import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/kubernetes/pkg/api/legacyscheme"
-	"k8s.io/kubernetes/pkg/apis/apps"
-	"k8s.io/kubernetes/pkg/apis/batch"
-	kapi "k8s.io/kubernetes/pkg/apis/core"
-	"k8s.io/kubernetes/pkg/apis/extensions"
 	"k8s.io/kubernetes/pkg/kubectl/polymorphichelpers"
 
-	appsapiv1 "github.com/openshift/api/apps/v1"
-	appsapi "github.com/openshift/origin/pkg/apps/apis/apps"
+	oappsv1 "github.com/openshift/api/apps/v1"
 )
 
 func NewUpdatePodSpecForObjectFn(delegate polymorphichelpers.UpdatePodSpecForObjectFunc) polymorphichelpers.UpdatePodSpecForObjectFunc {
 	return func(obj runtime.Object, fn func(*corev1.PodSpec) error) (bool, error) {
 		switch t := obj.(type) {
-		case *appsapi.DeploymentConfig:
```

Comment:
I've noticed some other polymorphichelpers of ours still use internals, followup?

Reply:
> I've noticed some other polymorphichelpers of ours still use internals, followup?

Yeah, followup.  they pass for now.

Reply:
I've added note to myself to fix those later on.